Update README.id.md

Busterz opened the pull request:
- Addresses #7446 
- Cannot fix with logo because cannot find "assets/join-slack-team.png" in the assets folder

Busterz committed:
Merge pull request #1 from Roshanjossey/master

Update from source-master

Busterz committed:
Update Readme.id.md

- Addresses #7446 Fix slack URL in Indonesian translation
- No join slack logo because assets folder does not have "assets/join-slack-team.png"

Futuro212 commented:
For the asset problem use the `src="../assets/join-slack-team.png`.

This happens because the readme's of the translations are not in the root path. So there's a need to go to the parent directory of the translations and then go to the assets.

I hope this can help you :)

Busterz commented:
but i can't seem to find join-slack-team.png manually in the assets folder either. Or maybe I am looking at somewhere wrong?

Busterz commented:
Oh, my bad. My repo is not updated one, considering how I forked this repo during the previous hacktoberfest!

Busterz committed:
Merge pull request #2 from firstcontributions/master

Updated repo

Busterz committed:
Update README.id.md

Updated ID slack url with logo

Roshanjossey commented on translations/README.id.md:
```diff
@@ -137,7 +137,7 @@ Hal ini sebenarnya tidak diperlukan namun nama dari branch ini menunjukkan tujua
 Mari rayakan kontribusi anda dan bagikan pengalaman tersebut bersama teman-teman dengan pergi ke [web app](https://roshanjossey.github.io/first-contributions/#social-share).
 
-Anda dapat bergabung dengan tim slack kami jika membutuhkan bantuan atau memiliki pertanyaan. [Bergabung dengan tim slack](https://firstcontributions.herokuapp.com).
+Anda dapat bergabung dengan tim slack kami jika membutuhkan bantuan atau memiliki pertanyaan. [<img align="right" width="150" src="../assets/join-slack-team.png">](https://join.slack.com/t/firstcontributors/shared_invite/enQtMzE1MTYwNzI3ODQ0LTZiMDA2OGI2NTYyNjM1MTFiNTc4YTRhZTg4OWZjMzA0ZWZmY2UxYzVkMzI1ZmVmOWI4ODdkZWQwNTM2NDVmNjY).
```
We don't want the image here. Here you can just change the link. We want the image changed at the top of the document. Refer the main Readme.

Roshanjossey commented:
Thank you so much for helping out @Futuro212 🖖 

@Busterz, please address the review comment.

Futuro212 commented:
You're welcome @Roshanjossey.

I did the same thing for #7898 :) and the pull request is open.

Busterz commented:
Oh, alright, I get what you mean. I am gonna make a change.

Busterz committed:
Update README.id.md

- Changed the link for "Bergabung dengan tim slack"
- Added "join slack team logo" at the top with the correct slack URL

Busterz commented:
Hi @Futuro212 and @Roshanjossey , I have changed accordingly. Please review it again! By the way, thank you for this experience!

Busterz committed:
Merge pull request #3 from firstcontributions/master

Update from resource

Busterz committed:
Update README.pt-pt.md

- Changed the link for "Junta-te à nossa equipa no Slack"
- Added "join slack team" logo with its corresponding slack url

Busterz committed:
Update README.pt-pt.md

- Changed the "join slack team" logo img src

Busterz committed:
Update README.lt.md

- Changed the link for "Prisijungti prie slack"
- Added "join slack team" logo and its corresponding slack URL

Roshanjossey commented on translations/README.lt.md:
```diff
@@ -1,5 +1,5 @@
 [![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.svg?v=103)](https://github.com/ellerbrock/open-source-badges/)
-[<img align="right" src="https://firstcontributions.herokuapp.com/badge.svg">](https://firstcontributions.herokuapp.com)
+[<img align="right" width="150" src="assets/join-slack-team.png">](https://join.slack.com/t/firstcontributors/shared_invite/enQtMzE1MTYwNzI3ODQ0LTZiMDA2OGI2NTYyNjM1MTFiNTc4YTRhZTg4OWZjMzA0ZWZmY2UxYzVkMzI1ZmVmOWI4ODdkZWQwNTM2NDVmNjY)
```
../assets

Busterz replied:
Hi, updated. Sorry for the blunder. I have checked the rest and it is the only one with this error. Please Review again!

Roshanjossey reviewed (changes requested):
Please make sure that your changes are correct before committing.

Busterz committed:
Update README.lt.md

Busterz committed:
Merge pull request #4 from firstcontributions/master

Update from resource

Busterz committed:
Update README.lt.md

Busterz commented:
Hi @Roshanjossey , I have edited the blunder that I did previously. Please review again! Thank you for your time!

Roshanjossey commented:
Thank you for making this change 🖖 ✨
